pom.xml

```diff
@@ -34,7 +34,8 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <appengine.api.version>1.9.40</appengine.api.version>
-    <docker.label.prefix>async-</docker.label.prefix>
+    <docker.tag>8-jre</docker.tag>
```

Comment:
I see your point about potential difficulty in working with timestamps as versions. So, we should discuss possible workarounds or push back on the whole idea. Maybe we should use something like git commit SHA instead.